app/views/layouts/_navbar.html.erb

```diff
@@ -43,7 +43,7 @@
           <li><%= link_to("Settings", "/users/"+current_user.id.to_s+"/edit", class: "navbar-menu__options", :id => current_user) %></li>
           <li><%= link_to("Edit Your Phenotypes", "/users/"+current_user.id.to_s+"/edit#phenotypes", class: "navbar-menu__options")%></li>
           <li class="divider"></li>
-          <li><%= link_to("Messages ("+current_user.messages.where(:user_has_seen => false).length.to_s+")","/users/" + current_user.id.to_s + "#messages", :method => "get", class: "navbar-menu__options")%></li>
+          <li><%= link_to "Messages (#{current_user.messages.where(:user_has_seen => false).length})","/users/#{current_user.id}#messages", class: 'navbar-menu__options', :data => { :reload => true }%></li>
```

Comment:
`"/users/#{current_user.id}#messages"` can be written as `user_path(current_user, anchor: 'messages')`. And I suppose the `:data => { :reload => true }` might have something to do with the reloading? :smile:

Reply:
Yeah the :method => "get" causes an endless reload on my end (right now on production too), Firefox 62.0.3 - when I force the reload then the entire page reloads, but only once! and it lands on the correct tab

Reply:
Which `:method => "get"`? I don't think it has anything to do with the link. It happens if you put `https://www.opensnp.org/users/15#messages` into your address bar as well. [This](https://github.com/openSNP/snpr/blob/a96b015fcfa1bed6aa23d33739beeceb67d55741/app/assets/javascripts/application.js#L51-L57) looks a bit suspicious.

Reply:
By method => get I mean the current way it's implement on production - 
That JS snippet is definitely the main 'suspect', getting stuck in an endless loop - let me fix that. My forced reloading seems to not have triggered that JS, thereby circumventing it (on my system?)

Reply:
I think what's missing in the JS is comparing `hash` with `anchor` and only redirecting if they don't match.

Reply:
Ah, no. Scratch that. It needs to figure out, whether the tab is active.

Reply:
What's weird is, that it only does that on the messages tab in the first place. :thinking:

Reply:
and why does this break only know? I put it in a year ago :P

Reply:
I have no idea. 😅

Reply:
This is how to do the tab thing properly, according to the [docs](https://getbootstrap.com/docs/3.3/javascript/#tabs).

```js
$('#myTabs a[href="#profile"]').tab('show')
```

Reply:
I've put in a new commit which gets rid of the method: get, and now it seems to work, even with the 'old' JS?